tests/app/test_retriever.py

```diff
@@ -396,6 +398,16 @@ def test_latest_message_from_each_thread_chosen_desc(self):
                     self.assertEqual(date[x], str(thread.all()[0].events[0].date_time))
                     self.assertEqual(msg_ids[x], thread.all()[0].events[0].msg_id)
 
+    def test_thread_count_by_survey(self):
+        """checks that the returned thread count is the same for every internal user"""
+        self.create_threads(5)
+
+        with self.app.app_context():
+            with current_app.test_request_context():
+                thread_count_internal = Retriever().thread_count_by_survey(self.user_internal, BRES_SURVEY)
+                thread_count_second_internal = Retriever().thread_count_by_survey(self.user_internal, BRES_SURVEY)
```

Comment:
Shouldn't this be self.second_user_internal?

Reply:
Yep, it really should be.  Changing it now.